editor.planx.uk/src/pages/FlowEditor/lib/store/editor.ts

```diff
@@ -482,6 +490,10 @@ export const editorStore: StateCreator<
 
   isFlowPublished: false,
 
+  isTemplate: false,
+
+  isTemplatedFrom: false,
```

Comment:
I'm assuming that in the _editor_, we only care about knowing this flow has been templated from another, not specifically _which one_ it's been templated from. That'll come into play later in API-side changes when we set up publishing triggers, etc. 

Any objections?

Reply:
It's a good question - it seems like it might be handy to have the ID - e.g. we need a title or to query something. I don't think we have a specific use case just now - no issues with a boolean 👍